conform-pr/main.go

```diff
@@ -28,52 +28,62 @@ func main() {
 		action.Fatalf("main: %s", err)
 	}
 
-	errors := runChecks(action, client)
-
-	if len(errors) == 0 {
-		return
+	summaries := runChecks(action, client)
+	action.AddStepSummary("| Check  | Status |")
+	action.AddStepSummary("|--------|--------|")
+
+	for _, summary := range summaries {
+		statusSign := ":heavy_multiplication_x:"
+		if summary.Ok {
+			statusSign = ":heavy_check_mark:"
```

Comment:
Yeah, we have a PR where we can do bikeshedding (and I'm serious about it!) 🤓

`:heavy_multiplication_x:` is ✖️  (actually, this one is for multiplication operation I believe)
`:heavy_check_mark:` is ✔️ 

Both are almost black and almost invisible in the dark mode. 

From the inclusivity perspective, should we use more colorful signs?

For example, these ones are [from the original article](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/):
`:x:`  - ❌ 
`:white_check_mark:` -  ✅

Reply:
whatever you want even if it's not ok for a light color scheme.

Reply:
WDYM?

Reply:
From my perspective (dark mode user) all emotes mentioned above are visible. I'm not sure if there's any differences between browsers, maybe it's a case?

Reply:
<img width="688" alt="Screenshot 2022-06-29 at 16 09 20" src="https://user-images.githubusercontent.com/1684673/176457661-8b6c2002-6d0a-4d83-bc03-81ae521b46db.png">

Darm mode proof

Reply:
![image](https://user-images.githubusercontent.com/42322511/176457916-4e752baa-a5d7-43e2-a3df-1273c832c5bb.png)
As I can see there are a few dark themes in the github settings

Reply:
done.

Reply:
Hm, the white checkmark looks ugly indeed.

Reply:
I actually, remembered that one I had to use 🍏 and 🍎 as those were the ones that were truly green and red :)

Reply:
![Screenshot from 2022-06-29 17-28-10](https://user-images.githubusercontent.com/4604088/176461782-3bfa9110-e077-494a-bc1d-56eb78c2f77b.png)

Reply:
🍏 and 🍎 are not the best choice for people with deuteranopia which is the most common color blindness form

(goes back to his cave)

Reply:
**Test** ✅

Reply:
✔️

Reply:
Well, if everything is fine, then why we added red and green apples?

Reply:
> Well, if everything is fine, then why we added red and green apples?

I added them, don't you see? 
Just need to test and github failure happened.

Reply:
Let's rollback apples and find a better approach later.